docker-entrypoint.sh

```diff
@@ -91,7 +91,7 @@ docker_init_database_dir() {
 	eval 'initdb --username="$POSTGRES_USER" --pwfile=<(echo "$POSTGRES_PASSWORD") '"$POSTGRES_INITDB_ARGS"' "$@"'
 
 	# unset/cleanup "nss_wrapper" bits
-	if [ "${LD_PRELOAD:-}" = '/usr/lib/libnss_wrapper.so' ]; then
+	if [ "${LD_PRELOAD:-}" = "$wrapper" ]; then
```

Comment:
Ah, great catch -- however, `wrapper` is only set conditionally above, so I think this fix needs to be a little more advanced.  We probably need to move `local wrapper` outside the `if` statement, and probably explicitly set it to the empty string (`local wrapper=''`) so that we can detect it appropriately down here:

```suggestion
	if [ -n "$wrapper" ] && [ -s "$wrapper" ] && [ "${LD_PRELOAD:-}" = "$wrapper" ]; then
```

Reply:
Actually,  it might even be easier then (since LD_PRELOAD should be empty anyway and we're unsetting it afterwards)
``` bash
if [ "w${LD_PRELOAD}" != "w" ]; then
```
or even simpler:
``` bash
if [ "${LD_PRELOAD}" != "" ]; then
```
or POSIX - the best in my opinion:
``` bash
case "$LD_PRELOAD" in %libnss_wrapper.so)
	# do stuff
	;;
esac
```

Reply:
Ah, good point -- however, this script is already Bash (and uses several bashisms), so something like this is probably the simplest option:

```suggestion
	if [[ "${LD_PRELOAD:-}" == */libnss_wrapper.so ]]; then
```

We could go overboard and also check for the two variables we immediately use being set as well, but I don't think that's really strictly necessary.

Reply:
Okay, I've updated it.